docs/installation/production/additional/official_docker.rst

```diff
@@ -313,9 +312,9 @@ You can overwrite the environment variable of the tethys base image in your app
 
 ::
 
-    ENV ASGI_PROCESSES 1
+    ENV ASGI_PROCESSES 4
 
-This line in your docker file will change the environment variable ASGI_PROCESSES from the default value of 4 to 1.
+This line in your docker file will change the environment variable ASGI_PROCESSES from the default value of 1 to 4.
```

Comment:
Show an example of enabling the Redis Channel layer with multiple ASGI_PROCESSES when using the Docker image.